kong/db/dao/routes.lua

```diff
@@ -0,0 +1,83 @@
+-- NOTE: this DAO is not enabled when router_flavor = expressions, see schema/entities/routes.lua
+
+
+local get_atc = require("kong.router.atc_compat").get_atc
+local constants = require("kong.constants")
+
+
+local _Routes = {}
```

Comment:
Just an FYI: Enterprise has an existing custom DAO for routes, so you will likely need to solve some conflicts when merging this over there.